interfaces/builtin/opengl.go

```diff
@@ -54,7 +54,7 @@ const openglConnectedPlugAppArmor = `
 /var/lib/snapd/hostfs/usr/share/glvnd/egl_vendor.d/*nvidia*.json r,
 
 # Main bi-arch GL libraries
-/var/lib/snapd/hostfs/{,usr/}lib{,32,64,x32}/{,@{multiarch}/}{,nvidia*/}lib{GL,GLESv1_CM,GLESv2,EGL,GLX}.so{,.*} rm,
+/var/lib/snapd/hostfs/{,usr/}lib{,32,64,x32}/{,@{multiarch}/}{,nvidia*/}lib{GL,GLU,GLESv1_CM,GLESv2,EGL,GLX}.so{,.*} rm,
```

Comment:
So we need more nvidia GLU?

Reply:
Some of the CUDA sample programs attempt to access `libGLU.so.*` on the host system. This is the exact apparmor denial:
```
Aug 21 09:08:56 ijohnson-XPS-15-9560 audit[12839]: AVC apparmor="DENIED" operation="open" profile="snap.cuda-samples.cuda-samples" name="/var/lib/snapd/hostfs/usr/lib/x86_64-linux-gnu/libGLU.so.1.3.1" pid=12839 comm="imageDenoising" requested_mask="r" denied_mask="r" fsuid=1000 ouid=0
```

This happens with the `imageDenoising` sample, which can be run individually with
`cuda-samples imageDenoising`.